app/sprinkles/core/locale/ar/messages.php

```diff
@@ -57,11 +57,11 @@
         '@TRANSLATION' => 'سياسة الخصوصية',
         'DESCRIPTION'  => 'تحدد سياسة الخصوصية لدينا نوع المعلومات التي نجمعها منك وكيفية استخدامها.',
     ],
-    // 'SLUG'                     => 'Slug',
+    'SLUG'                     => 'سبيكة',
     'SLUG_CONDITION'           => 'Slug/الظروف',
```

Comment:
I wasn't sure if there is a comparable word for `slug`. If so, should this line be updated?